src/commands/deal/deploy.ts

```diff
@@ -180,31 +174,10 @@ export default class Deploy extends BaseCommand<typeof Deploy> {
         chainNetworkId,
       };
 
-      await deployedConfig.$commit();
+      await workersConfig.$commit();
     }
 
-    await writeFile(
-      await ensureFluenceAquaDealPath(),
-      jsToAqua(deployedConfig.workers, "getWorkersInfo"),
-      FS_OPTIONS
-    );
-
-    try {
-      const mainAquaFileContent = await readFile(
-        await ensureSrcAquaMainPath(),
-        FS_OPTIONS
-      );
-
-      await writeFile(
-        await ensureSrcAquaMainPath(),
-        mainAquaFileContent.replace(
-          MAIN_AQUA_FILE_STATUS_TEXT_COMMENT,
-          MAIN_AQUA_FILE_STATUS_TEXT
-        ),
-        FS_OPTIONS
-      );
-    } catch {}
-
+    await ensureAquaFileWithWorkerInfo(workersConfig);
     commandObj.log(`\nDeploy completed successfully`);
```

Comment:
Would be nice to outline what has been deployed. Some kind of status.

Reply:
let's come up with something. What can it be? The one thing that I absolutely can show is the stuff that I write to the new workers.yaml (which previously was deployed.yaml)

Reply:
> workers.yaml
Yeah, at least that, for now.



Later we will have to do some kind of monitoring for deployments. Like
```
Deploy completed successfully
Deal contract is: <color>0xabcacbacbacbacbacba</color>

Do you want to monitor providers accepting the deal? y/n
y
Waiting for providers to provide collateral... (100 USDC per a single worker)

12D3ABCACBACBACXYZ: 100 USDC
12D3UUUUUAAAAHHHHA: 200 USDC
12D3BYBYBYBYBYBYBY: 300 USDC
```
And then maybe even give a way to retrieve logs of Installation Spells of workers on these providers